FEXCore/Source/Interface/Core/OpcodeDispatcher/Flags.cpp

```diff
@@ -200,13 +200,39 @@ OrderedNode *OpDispatchBuilder::GetPackedRFLAG(uint32_t FlagsMask) {
   return Original;
 }
 
-void OpDispatchBuilder::CalculateOF_Add(uint8_t SrcSize, OrderedNode *Res, OrderedNode *Src1, OrderedNode *Src2) {
+void OpDispatchBuilder::CalculateOF(uint8_t SrcSize, OrderedNode *Res, OrderedNode *Src1, OrderedNode *Src2, bool Sub) {
```

Comment:
The definition in the header here for `bool Sub` is `bool Add`.
Which one is it?

Reply:
Sub, this was a typo. Thanks for spotting that, and fixed!